MdeModulePkg/Core/Dxe/Dispatcher/Dispatcher.c

```diff
@@ -869,7 +875,13 @@ CoreAddToDriverList (
   // NULL or FALSE.
   //
   DriverEntry = AllocateZeroPool (sizeof (EFI_CORE_DRIVER_ENTRY));
-  ASSERT (DriverEntry != NULL);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (DriverEntry == NULL) {
+    ASSERT (DriverEntry != NULL);
+    return EFI_OUT_OF_RESOURCES;

```

Comment:
Should add the status type as a `@retval`. I know this is poorly followed and "others" is often used to umbrella different values. Since this is DXE Core though, which is highly depended upon, it would be nice to keep the list updated.